src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java

```diff
@@ -233,18 +285,18 @@ private void addNewSlave(final AmazonEC2 ec2, final String instanceId) throws Ex
             return; // Wait some more...
 
         final FleetNode slave = new FleetNode(instanceId, "Fleet slave for" + instanceId,
-                fsRoot, "1", Node.Mode.NORMAL, "ec2-fleet", new ArrayList<NodeProperty<?>>(),
+                fsRoot, "1", Node.Mode.NORMAL, this.labelString, new ArrayList<NodeProperty<?>>(),
```

Comment:
Is there any particular reason why the number of executors is set to "1"?
I'd like to run multiple jobs inside of Docker containers on each slave instance. 
I can only assume that the reason in the logic behind fleet scaling.